SPECS/kernel/config_aarch64

```diff
@@ -374,6 +374,7 @@ CONFIG_ARM64_ERRATUM_2457168=y
 CONFIG_ARM64_WORKAROUND_TSB_FLUSH_FAILURE=y
 CONFIG_ARM64_ERRATUM_2054223=y
 CONFIG_ARM64_ERRATUM_2067961=y
+CONFIG_ARM64_ERRATUM_3194386=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/cc627930b2ac5dad7a5d45d3052e0f057b27e4b3